### PR TITLE
workaround pysankey issue and enhanced button for next question

### PR DESCRIPTION
**Pull request by gjelsas**
#I added a workaround for the pySankey != pysankey import error and added some code for the differentiation between the next question and the last question which closes the survey. For issue #117 and #58

### CODE REVIEW

**Comment by gjelsas** [comment]
Update for the German translation.

File: survey/exporter/tex/question2tex_sankey.py [hunk 1]
```diff
@@ -4,7 +4,10 @@
 
 from django.utils.translation import gettext_lazy as _
 from pandas.core.frame import DataFrame
-from pysankey import sankey
+try: 
+    from pysankey import sankey
```
Comment (Pierre-Sassoulas) [inline survey/exporter/tex/question2tex_sankey.py:8]: Looking at the dependencies it seems that it should work (we use ``["pySankeyBeta~=1.2.2"]``).Maybe the fix here would be:

```python
try:
    from pysankey import sankey
except ModuleNotFoundError:
    warnings.warn(
        "Cannot import sankey, please install the package using the sankey extra."
        " (pip install django-survey-and-report[sankey])"
    )
```

Or am I missing something ? Did you install that way and still encounter the problem ?

**Review by Pierre-Sassoulas — APPROVED** [review]
Looks good, thank you for that ! I just have a minor question regarding the problem with sankey.

File: survey/exporter/tex/question2tex_sankey.py (hunk 1 above)
Comment (gjelsas) [reply survey/exporter/tex/question2tex_sankey.py:8]: No, you are absolutely right. I had it wrong in my virtualenv... That patch is not necessary anymore. 
Thanks for your work on this!

**Comment by Pierre-Sassoulas** [comment]
Thank you for the contribution, I'm going to release a new version with those changes shortly.

**Pull request merged**